BackendBench/eval.py

```diff
@@ -31,34 +39,119 @@ def format_exception(e, op, args, kwargs):
     return EXC_MSG.format(op=op_name, args=serialize_args(args, kwargs), exc=e)
 
 
-def allclose(a, b):
+def _allclose(a, b):
```

Comment:
in the PR description do you mind describing in longer form what all these changes are about and how you're confident that previous semantics didn't change, otherwise this is a bit tricky to review